qsignature/src/org/qcmg/sig/SignatureCompareRelatedSimpleGenotypeMT.java

```diff
@@ -37,13 +34,18 @@
 import org.qcmg.sig.util.ComparisonUtil;
 import org.qcmg.sig.util.SignatureUtil;
 
+import gnu.trove.map.hash.THashMap;
+import gnu.trove.map.hash.TIntByteHashMap;
+
 /**
  * This class gets a list of all .qsig.vcf files from the supplied path.
  * It then performs a comparison between them all, regardless of whether they are bam or snp chip files
  * An xml output file is produced
  * If any comparison scores are less than the cutoff, they are added to a list, which is then emailed to interested parties informing them of the potential problem files
  *  
  * @author o.holmes
+ * 
+ * @Deprecated Superseded by Compare
```

Comment:
again, deprecated now if it is planed to delete on next version

Reply:
done